tests/integration/raw_studies_blueprint/test_fetch_raw_data.py

```diff
@@ -297,206 +298,405 @@ def test_get_study(self, client: TestClient, user_access_token: str, internal_st
+        study_id = res.json()
+
+        # add a new area to the study
         res = client.post(
-            f"/v1/studies/{internal_study_id}/copy",
-            headers={"Authorization": f"Bearer {user_access_token}"},
-            params={"dest": "default", "with_outputs": False, "use_task": False},
+            f"/v1/studies/{study_id}/areas",
+            json={
+                "name": "area 1",
+                "type": "AREA",
+                "metadata": {"country": "FR", "tags": ["a"]},
+            },
         )
+        assert res.status_code == 200, res.json()
+
+        # archive the study
+        res = client.put(f"/v1/studies/{study_id}/archive")
+        assert res.status_code == 200
+        task_id = res.json()
+        wait_for(
+            lambda: client.get(
+                f"/v1/tasks/{task_id}",
+            ).json()["status"]
+            == 3
+        )
+
+        # retrieve a `Desktop.ini` file from inside the archive
+        rel_path = "Desktop"
+        res = client.get(
+            f"/v1/studies/{study_id}/raw",
+            params={"path": rel_path, "formatted": True},
+        )
+        assert res.status_code == 200
+
+        # retrieve a `study.antares` file from inside the archive
+        rel_path = "study"
+        res = client.get(
+            f"/v1/studies/{study_id}/raw",
+            params={"path": rel_path, "formatted": True},
+        )
+        assert res.status_code == 200
+
+
+@pytest.mark.integration_test
+class TestFetchOriginalFile:
+    """
+    Check the retrieval of a file from Study folder
+    """
+
+    def test_get_study_file(
+        self,
+        client: TestClient,
+        user_access_token: str,
+        internal_study_id: str,
+    ):
+        """
+        Test the `get_study_file` endpoint for fetching for a file in its original format.
+
+        This test retrieves a specific file from a study identified by a UUID and checks
+
+        The test performs the following steps:
+        1. Copies the user resources in the Study directory.
+        2. Uses the API to download a file from the "user/folder" directory.
+        3. Compares the fetched data with the expected file from disk.
+        4. Check for cases where Errors should be returned.
+        """
+        # First copy the user resources in the Study directory
+        with db():
+            study: RawStudy = db.session.get(Study, internal_study_id)
+            study_dir = pathlib.Path(study.path)
+        client.headers = {"Authorization": f"Bearer {user_access_token}"}
+        original_file_url = f"/v1/studies/{internal_study_id}/raw/original-file"
+
+        shutil.copytree(
+            ASSETS_DIR.joinpath("user"),
+            study_dir.joinpath("user"),
+            dirs_exist_ok=True,
+        )
+
+        # Then, use the API to download the files from the "user/folder" directory
+        user_folder_dir = study_dir.joinpath("user/folder")
+        for file_path in user_folder_dir.glob("*.*"):
```

Comment:
Don't see the point of this part, you can remove it IMO

Reply:
it is the API test, so you retrieve a file via the API and verify that the bytes content match with what you have on disk

Reply:
you do this for many files so you know there is no a specific extension that does not work